camera_calibration/package.xml

```diff
@@ -21,7 +21,7 @@
   <run_depend>std_srvs</run_depend>
   <run_depend>sensor_msgs</run_depend>
   
-  <exec_depend>python_orocos_kdl</exec_depend>
+  <run_depend>python_orocos_kdl</run_depend>
 
```

Comment:
Why did you change it here? exec_depend is new version of run_depend...

Reply:
addressed